Update README.md

christoc-ibm opened the pull request:
Added Cloud Foundry deprecation notice

christoc-ibm committed:
Update README.md

Added Cloud Foundry deprecation notice

nfstein reviewed (changes requested):
We dont deploy vapor to CF, please remove any references to CF we still have in there

christoc-ibm committed:
Update README.md

Removed CF deprecation statement b/c this skit doesn't deploy to CF.
Also removed the CF deploy command that was incorrectly in this readme.

christoc-ibm commented:
Removed all CF references.

nfstein commented on README.md:
```diff
@@ -79,7 +79,6 @@ ibmcloud dev run
 Your application is launched locally. When you are ready to deploy to IBM Cloud on Cloud Foundry or Kubernetes, run one of the following commands:
```
one here

christoc-ibm replied:
Dang! Good catch. Removed.

christoc-ibm committed:
Update README.md
